### PR TITLE
small improvements to logs

### PR DESCRIPTION
**Pull request by james-prysm**
**What type of PR is this?**

Other

**What does this PR do? Why is it needed?**

removes some duplicate logs, or moves them to be less redudnant

**Which issues(s) does this PR fix?**

Fixes #

**Other notes for review**

**Acknowledgements**

- [x] I have read [CONTRIBUTING.md](https://github.com/prysmaticlabs/prysm/blob/develop/CONTRIBUTING.md).
- [x] I have made an appropriate entry to [CHANGELOG.md](https://github.com/prysmaticlabs/prysm/blob/develop/CHANGELOG.md).
- [x] I have added a description to this PR with sufficient context for reviewers to understand this PR.

### CODE REVIEW

File: validator/client/service.go [hunk 1]
```diff
@@ -292,9 +292,6 @@ func ConstructDialOptions(
 		transportSecurity = grpc.WithTransportCredentials(creds)
 	} else {
 		transportSecurity = grpc.WithInsecure()
-		log.Warn("You are using an insecure gRPC connection. If you are running your beacon node and " +
```
Comment (james-prysm) [inline validator/client/service.go:295]: redundant log

File: config/proposer/loader/loader.go [hunk 2]
```diff
@@ -159,6 +155,10 @@ func (psl *settingsLoader) Load(cliCtx *cli.Context) (*proposer.Settings, error)
 			if psl.options.builderConfig != nil {
 				defaultConfig.Builder = psl.options.builderConfig.ToConsensus()
 			}
+			if psl.existsInDB && len(psl.loadMethods) == 1 {
+				// only log the below if default flag is the only load method
```
Comment (james-prysm) [inline config/proposer/loader/loader.go:159]: log is a bit noisy, mostly debugging purposes

File: validator/client/validator.go [hunk 3]
```diff
@@ -1327,6 +1327,11 @@ func (v *validator) buildSignedRegReqs(
 	if v.genesisTime > uint64(time.Now().UTC().Unix()) {
 		return signedValRegRequests
 	}
+
+	if v.ProposerSettings().DefaultConfig != nil && v.ProposerSettings().DefaultConfig.FeeRecipientConfig == nil && v.ProposerSettings().DefaultConfig.BuilderConfig != nil {
```
Comment (james-prysm) [inline validator/client/validator.go:1331]: printing too much in loop

**Review by terencechain — APPROVED** [review]

File: validator/client/service.go (hunk 1 above)
Comment (james-prysm) [reply validator/client/service.go:295]: putting it back in, fails test + was supposed to do some more setup

**Review by terencechain — APPROVED** [review]

**Pull request merged**